components/base-war/src/main/webapp/templates/macros.vm

```diff
@@ -1418,7 +1418,13 @@ ${Msz}Mb##
     #if ("$!resultPage" == '')
       #set ($resultPage = 'XWiki.LiveTableResults')
     #end
-    #set ($dataurl = $xwiki.getURL($resultPage, 'get', "$escapetool.url($parameters)&$!options.extraParams"))
+    #if ($resultPage.startsWith('/'))
+       #set ($hostname = "$request.getServerName()")
+       #set ($hostport = "$request.getServerPort()")
+       #set ($dataurl = "http://$hostname:$hostport$resultPage?$escapetool.url($parameters)&$!options.extraParams")
```

Comment:
Wdym? this line shoulf simply read `#set ($dataurl = "$resultPage?$escapetool.url($parameters)&$!options.extraParams")`